packages/agents/sequencer/src/lib/operations/relayer/fastpath.ts

```diff
@@ -30,6 +30,17 @@ export const sendExecuteFastToRelayer = async (
 
   const executeEncodedData = await encodeExecuteFromBids(round, bids, transfer, requestContext);
 
+  // Simulation data for Execute transfer
+  const relayerFrom = await relayers[0].instance.getRelayerAddress(destinationChainId);
```

Comment:
Not ideal lol. We could just use the hardcoded Gelato address here. It will always work on every chain.

Reply:
we won't have gelato for every chain and ideally, this shouldn't be an issue with latency too. We only need a from address which is whitelisted.

Reply:
Yes we don't have gelato but we still whitelist their relayer address.

Reply:
resolved